packages/dev/gui/src/2D/controls/stackPanel.ts

```diff
@@ -165,7 +165,7 @@ export class StackPanel extends Container {
                     child._top.ignoreAdaptiveScaling = true;
                 }
 
-                if (child._height.isPercentage && !child._automaticSize) {
+                if (child._height.isPercentage && !child._automaticSize && !(child as any)._resizeToFit) {

```

Comment:
nit - can we avoid using any? TextBlock is already imported as type, this should work as is
```suggestion
                if (child._height.isPercentage && !child._automaticSize && !(child as TextBlock)._resizeToFit) {
```